packages/main/src/Menu.ts

```diff
@@ -269,6 +270,10 @@ class Menu extends UI5Element {
 		return this.items.filter((item): item is MenuItem => !item.isSeparator);
 	}
 
+	get menuAccessibleName() {
```

Comment:
do we have any other accessible name? As we call the getter "menuAccessibleName" when we are in the menu file. I would suggest something like "acessibleNameText"

Reply:
done